html/js/tree/Tree.js

```diff
@@ -12,34 +12,45 @@ GTE.TREE = (function (parentModule) {
     }
 
     /**
-    * Function that draws the Game in the global canvas starting from a node
-    * Takes care of updating the positions, clearing the canvas and drawing in it
+    * Function that draws the Game in the global canvas starting from an information set
+    * Takes care of updating the positions, clearing the drawing and redrawing in it
     */
     Tree.prototype.draw = function(){
         if (!this.positionsUpdated) {
             this.updatePositions();
         }
-        GTE.canvas.clear();
+        this.clear();
         this.recursiveDraw();
     };
 
     /**
-    * Recursive function that draws the Game in the global canvas starting from a node
-    * If no param is given it will start from root
+    * Function that takes care of clearing the canvas and deleting all labels
+    */
+    Tree.prototype.clear = function(){
+        // Clear canvas
+        GTE.canvas.clear();
+    };
+
+    /**
+    * Recursive function that draws the Game in the global canvas
+    * starting from a given iSet. If no param is given it will
+    * start from root.
     * Stopping criteria: that the current node is a leaf
     * Recursive expansion: to all of the node's children
-    * @param {Node} [node] Node to start drawing from
+    * @param {ISet} [iSet] ISet to start drawing from
```

Comment:
Same thing. In this case, if iSet is undefined it will be set to the root, so we can say it is an optional parameter, therefore the brackets make sense.